brh.data-commons.org/metadata/aggregate_config.json

```diff
@@ -43,8 +43,8 @@
       }
     },
     "JCOIN": {
-      "mds_url": "https://jcoin.datacommons.io/",
-      "commons_url": "jcoin.datacommons.io/",
+      "mds_url": "https://jcoin.datacommons.io",
+      "commons_url": "jcoin.datacommons.io",
```

Comment:
There is one more place in this file which has a trailing `/` (`Cancer Imaging Data Commons`). I wonder why do we have certain urls with a trailing `/` and a few urls which don't?

* Could this have been a typo?
* If yes can we fix both of these under a single PR?

Reply:
Good catch, let me push another commit to address that.